adapters/ion-adapter/src/main/java/com/connexta/ion/replication/adapters/ion/IonNodeAdapter.java

```diff
@@ -150,6 +152,18 @@ public void close() throws IOException {
     // nothing to close
   }
 
+  private HttpEntity getMetadataEntity(Metadata metadata) {
```

Comment:
The get entity methods were added to ensure the content types are set correctly.